magento/pwa-studio
# Add some slightly more verbose upgrade installation instructions

## Description

This PR introduces an update to the README, specifically in the `Upgrading from a previous version` section.

With this update, upgrading is no longer something that takes too much cognitive effort, as the `package.json` file will be automatically updated for users.

By using the `yarn add ...` method, it's less likely that bugs will be introduced into the project by someone having to manually edit version numbers in the `package.json` file. Also, it removes the need for people to go hunting for the latest version of each package to manually put into the `package.json` file.

The one caveat is if `@magento` has some new packages added, then the sample command will need to be updated between releases, rather than just left as is.

## Related Issue
Closes #2661.

## Acceptance 



### Verification Stakeholders

### Specification



### Verification Steps
1. Open README
2. Confirm that upgrade instructions are clearer

## Checklist

- [c] I have added tests to cover my changes, if necessary.
